### PR TITLE
RavenDB-17264 - removing the AGPL alert shouldn't prevent us from loading the license

### PR DESCRIPTION
**Pull request by grisha-kotler**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17264

### Additional description

Removing the AGPL alert shouldn't prevent us from loading the license

### Type of change

- Optimization

### How risky is the change?

- Low 

### Testing 

- It has been verified by manual testing

### CODE REVIEW

File: src/Raven.Server/Commercial/LicenseManager.cs [hunk 1]
```diff
@@ -242,7 +241,15 @@ private void CreateAgplAlert()
...
         {
-            _serverStore.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.LicenseManager_AGPL3, null));
+            try
+            {
+                _serverStore.NotificationCenter.Dismiss(AlertRaised.GetKey(AlertType.LicenseManager_AGPL3, null));
+            }
+            catch (Exception e)
+            {
+                // nothing we do can here, we'll try to remove it on next restart or when reloading the license
+                Logger.Info("Failed to remove the AGPL alert", e);
```
Comment (ayende) [inline src/Raven.Server/Commercial/LicenseManager.cs]: What scenario did you see that caused this?

Also `if (Logger.IsOperationsEnabled)` and log that as `Operations`

Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: After starting the server and during license initialization (or when the license is renewed - every day), we tried to open a write tx in order to remove the AGPL notification.
The write was already taken by a cluster operation (global notifications storage is using the server store).

Comment (ayende) [reply src/Raven.Server/Commercial/LicenseManager.cs]: Wouldn't we wait until this is completed?

Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: It throws `ThrowOnTimeoutWaitingForWriteTxLock` after 30 seconds.
The lock is held by `Follower thread from Remote connection` (we observed slow IO during that time).

Comment (grisha-kotler) [reply src/Raven.Server/Commercial/LicenseManager.cs]: Pushed my changes. We'll skip deleting the notification when it doesn't exist (on a running it shouldn't exist anyway).

**Pull request merged**